mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -63,7 +63,7 @@ def get_supported_targets():
         except AttributeError:
             try:
                 # this should works for >=v1.3.0
-                list_supported_targets = sorted(set(list(mbeds.plat_db._dbs.values())[0].values()))
+                list_supported_targets = sorted(set(list(mbeds.plat_db.items().values())[0].values()))
```

Comment:
That probably does not do what you want. That'll always produce an `AttributeError` when trying to get the `values` method on the `items` iterator.

Reply:
ouch true. need something else then..

Reply:
You want:
```python
sorted(set(name for id, name in mbeds.plat_db.items()))

Reply:
thanks, that was exactly what I looked for :)

Reply:
I'm actually considering making that the official API. That has the advantage that you should be able to correctly use mocked targets.